polyfills/Object.prototype.toString/polyfill.js

```diff
@@ -0,0 +1,6 @@
+var Object_prototype_toString = Object.prototype.toString
+Object.prototype.toString = function() {
+    if (this === null) return "[object Null]";
+    if (this === undefined) return "[object Undefined]";
+    return Object_prototype_toString.call(this);
+}
```

Comment:
I suggest returning the new function from an IIFE into which you pass the existing native impl to avoid creating a variable that exists across all polyfills.